proto/eth/service/beacon_chain_service.proto

```diff
@@ -51,9 +51,11 @@ service BeaconChain {
     option (google.api.http) = { get: "/internal/eth/v1/beacon/genesis" };
   }
 
+  // DEPRECATED: GetWeakSubjectivity endpoint will no longer be supported
   // GetWeakSubjectivity is a new proposed endpoint to retrieve the details necessary to download
   // the ssz data needed to start a beacon node - checkpoint(epoch + block_root) and state_root
   rpc GetWeakSubjectivity(google.protobuf.Empty) returns (v1.WeakSubjectivityResponse) {
```

Comment:
For the sake of consistency, put the deprecated notice at the end of the comment.
```suggestion
  // GetWeakSubjectivity is a new proposed endpoint to retrieve the details necessary to download
  // the ssz data needed to start a beacon node - checkpoint(epoch + block_root) and state_root
  // DEPRECATED: GetWeakSubjectivity endpoint will no longer be supported
  rpc GetWeakSubjectivity(google.protobuf.Empty) returns (v1.WeakSubjectivityResponse) {
```